proxy/core/src/main/scala/io/cloudstate/proxy/eventsourced/EventSourcedEntity.scala

```diff
@@ -349,7 +352,6 @@ final class EventSourcedEntity(configuration: EventSourcedEntity.Configuration,
 
     case event: pbAny =>
       maybeInit(None)
-      relay ! EventSourcedStreamIn(EventSourcedStreamIn.Message.Event(EventSourcedEvent(lastSequenceNr, Some(event))))
```

Comment:
Why was this removed?

Reply:
@viktorklang may be i took the wrong. I removed it just to show how it should work for the the eventually new CrudEntity. It is removed because i think the User Function should not deal with events but only with snapshot. And in the future there will be EventSourcedEntity and CrudEntity.